pkg/pillar/cmd/vcomlink/client/client.go

```diff
@@ -15,7 +15,7 @@ func getTpmEkFromHost(fd int) error {
 		BasePacket: vcom.BasePacket{
 			Channel: vcom.ChannelTpm,
 		},
-		Request: vcom.TpmRequestGetEk,
+		Request: vcom.RequestTpmGetEk,
```

Comment:
This whole commit seems also a candidate for a squash. The rule of thumb is not to make fixes on commits on the same PR, otherwise difficult to review

Reply:
done.